cmd/opampsupervisor/supervisor/commander/commander.go

```diff
@@ -39,6 +40,8 @@ func NewCommander(logger *zap.Logger, cfg *config.Agent, args ...string) (*Comma
 		cfg:     cfg,
 		args:    args,
 		running: &atomic.Int64{},
+		doneCh:  make(chan struct{}, 1),
```

Comment:
I'm noticing that the channel is made buffered here, but in Start it's recreated as an unbuffered channel, I think it should be consistent (should the channel be initialized here or just in Start?)